onnx/numpy_helper.py

```diff
@@ -86,10 +86,11 @@ def from_array(arr, name=None):  # type: (np.ndarray[Any], Optional[Text]) -> Te
         tensor_def: the converted tensor def.
     """
     tensor = TensorProto()
-    tensor.dims.extend(arr.shape)
     if name:
         tensor.name = name
 
+    tensor.dims.extend(arr.shape)
```

Comment:
Is this an accidental change? Moving the line around doesn't seem to change anything.

Reply:
Yes, thanks. This is not intended.